includes/ampli/flush/ampli-flush-section.md

```diff
@@ -0,0 +1,9 @@
+### Flush
+
+Call `flush()` to immediately send any pending events.
+
+The Amplitude SDK queues events and sends them on an interval based on the configuration. By default, the SDK will flush
+every 10 events or 30 seconds, which ever happens first.
```

Comment:
Is that always 10 for the threshold? It might be for node just not sure if we reference it to others it would still be true.

Reply:
Thanks, I think I am going to remove the exact numbers from this section to make it more generic.